introduction/architecture/routers-architecture.md

```diff
@@ -2,4 +2,6 @@
 
 The overall view of the system architecture of CaraML Traffic Routers can be illustrated in the diagram below:
 
-![Routers architecture](../../.gitbook/assets/turing\_architecture.png)
+![Routers architecture](../../module/router/assets/turing_architecture.png)
```

Comment:
For this, we want to replace https://github.com/caraml-dev/docs/blob/main/.gitbook/assets/turing_architecture.png with the system architecture doc, yeah? I think that will make it more consistent with Models, Feature store, etc. Just confirming.

Reply:
yes, I changed the image to be using the system architecture diagram from the Turing readme. It should be referring to this image after the change https://github.com/caraml-dev/docs/blob/main/module/router/assets/turing_architecture.png